docs/content/contributing/terraform/terraform-contribution-flow/_index.md

```diff
@@ -35,11 +35,14 @@ flowchart TD
     click D "{{% siteparam base %}}/contributing/terraform/terraform-contribution-flow/#4-run-pre-commit-checks"
   E(5 - Create a pull request to the upstream repository)
     click E "{{% siteparam base %}}/contributing/terraform/terraform-contribution-flow/#5-create-a-pull-request-to-the-upstream-repository"
+  F(6 - Get your pull request reviewed & approved)
```

Comment:
```suggestion
  F(6 - Get your pull request reviewed, address any comments & have it approved)
```

Reply:
I prefer not to have such long chapter title and so many words in the flow chart. In fact, I shortened it even further, given the whole point here is to get the PR approved - all requirements need to be further detailed in the actual chapter. Therefore, I also added an info-box elaborating on the point you were trying to make (i.e., all outstanding comments need to be addressed and the module owner needs to make sure the publication pipeline succeeds).
<img width="345" height="159" alt="image" src="https://github.com/user-attachments/assets/174c1601-c798-4e1c-840c-046bf766ce53" />
<img width="1200" height="174" alt="image" src="https://github.com/user-attachments/assets/6271b1c3-eb7d-4a3e-8214-a3fe9fc2ff13" />